### PR TITLE
Chore: Removes Radio component

### PR DESCRIPTION
**Pull request by hellofanny**
Related to https://github.com/vtex/faststore/pull/1513

## What's the purpose of this pull request?

We aren't using `Radio` component in the project. And it has been moved to faststore/ui as [RadioField](https://github.com/vtex/faststore/pull/1513)

## How to test it?

Storybook won't list Radio component.

## References

<em>Spread the knowledge: is any content you used to create this PR worth sharing?</em>

<em>Extra tip: add references to related issues or mention people important to this PR may be good for the documentation and reviewing process</em>

## Checklist

<em>You may erase this after checking them all :wink:</em>

**PR Title and Commit Messages**
- [ ] PR title and commit messages follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification
  - Available prefixes: `feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, and `test`

**PR Description**
- [ ] Added a label according to the PR goal - `Breaking change`, `Features`, `Bug fixes`, `Chore`, `Documentation`, `Style changes`, `Refactoring`, `Performance`, and `Test`
- [ ] Added the component, hook, or path name in-between backticks (\`\`) - *if applicable, e.g., `ComponentName` component, `useWindowDimensions` hook*

**Dependencies**
- [ ] Committed the `yarn.lock` and `bun.lockb` file when there were changes to the packages

**Documentation**
- [ ] PR description
- [ ] Added to/Updated the Storybook - *if applicable*
- [ ] For documentation changes, ping `@carolinamenezes` or `@PedroAntunesCosta` to review and update

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1471,7 +1467,7 @@
     "@reach/popover" "^0.16.0"
     include-media "^1.4.10"
     modern-normalize "^1.1.0"
-    react-swipeable "^6.1.2"
+    react-swipeable "^7.0.0"
```
Comment (eduardoformiga) [inline yarn.lock:1470]: do we need to include the `yarn.lock` changes?

**Review by eduardoformiga — APPROVED** [review]
LGTM!

File: yarn.lock (hunk 1 above)
Comment (hellofanny) [reply yarn.lock:1470]: actually, I think we can just ignore heheh

**Comment by hellofanny** [comment]
> LGTM!
> 
> isn't it good for storybook purposes?

Maybe, but we decided to remove storybook from the core in the evergreen proposal. So I thought it was ok to remove it now since it's not being used anyway.

**Comment by eduardoformiga** [comment]
> > LGTM!
> > isn't it good for storybook purposes?
> 
> @eduardoformiga Maybe, but we decided to remove storybook from the core in the evergreen proposal. So I thought it was ok to remove it now since it's not being used anyway.

That's true. My bad! I had forgotten that hahah

**Pull request merged**